### DISCUSSION TITLE
Cmd commands?

### DISCUSSION THREAD

**Discussion by TheExiledCat** [AIRLegend/aitrack#186] [Ideas]
Is it possible yet to start the tracker and have it start tracking using a cmd Command or batch file? this would help alot for some players that use a batch file to start all their software at once

**Comment by searching46dof** [comment]
Adding a commandline option to start tracking may be the easiest way to implement this.

However, currently AiTrack manual brightness setting sets the exposure. When manual brightness is not selected, it is at a fixed exposure instead of autoexposure. Everytime I start, I need to manually adjust the brightness for the ambient lighting (dependent on time of day and weather) and amount of indoor lighting.

**Comment by chrismegyeri** [comment]
Please add this, been looking for command line syntax since I got it working, automating my launchers for DCS and Star Citizen would be super useful.

My suggestion would be to consider migrating all settings to a .ini file or XML file so items as covered with the previous can be addressed via scripts  "If statements" relating to time of day etc for cornerstone use cases, pretty easy to point a application launch to set configuration file with "Autostart" flags set via CLI.

This would allow users to do whatever settings dependent on conditions the user sets such as time of day, running processes etc.
